R/GeocodeIPs.R

```diff
@@ -29,10 +35,82 @@ GeocodeIPs <- function(ips) {
     if (!((is.character(ips) && is.null(dim(ips))) || is.factor(ips)))
         StopForUserError("Please provide a character vector of IP addresses.")
 
-    file <- system.file("extdata", "GeoLite2-Country.mmdb", package = "rgeolocate")
+    loadDatabase()
+    checkIP2LocationLibraryAvailable()
+    lapply(ips, get_all) |> standardiseGeocodeIPsColumns(ips = ips)
+}
+
+getIP2LocationDatabasePath <- function() {
+    db.path <- Sys.getenv("IP2LOCATION_DB_PATH", unset = NA)
+    if (is.na(db.path)) {
+        StopForUserError(
```

Comment:
Should these be `StopForUserError`? I think we would want to be notified about this, at least with a bugdup (when R errors go to bugdup).

Reply:
As JC pointed out, these will also be shown to users, so perhaps a more user-friendly message should be shown like "The was a problem with the IP address database. Please contact support if this is causing hardship. Error code: 123". Where the error code helps us identify the specific error.

Reply:
Updated in [67113ce](https://github.com/Displayr/flipAPI/pull/66/commits/67113ce17486ca5d5e301c43d66c4ec2732e5d92)